migrate/v0_11/migrate.go

```diff
@@ -65,3 +67,53 @@ func MigrateBep3(oldGenState v0_9bep3.GenesisState) v0_11bep3.GenesisState {
 		PreviousBlockTime: v0_11bep3.DefaultPreviousBlockTime,
 	}
 }
+
+// MigrateCDP migrates from a v0.9 (or v0.10) cdp genesis state to a v0.11 cdp genesis state
+func MigrateCDP(oldGenState v0_9cdp.GenesisState) v0_11cdp.GenesisState {
+	var newCDPs v0_11cdp.CDPs
+	var newDeposits v0_11cdp.Deposits
+	var newCollateralParams v0_11cdp.CollateralParams
+	newStartingID := uint64(0)
+
+	for _, cdp := range oldGenState.CDPs {
+		newCDP := v0_11cdp.NewCDPWithFees(cdp.ID, cdp.Owner, cdp.Collateral, "bnb-a", cdp.Principal, cdp.AccumulatedFees, cdp.FeesUpdated)
```

Comment:
This seems reasonable as block time picks up where it left off, but I haven't tested it yet. Have we confirmed that this results in fees being updated correctly without any minor math hiccups?

Reply:
Yes, I've confirmed that the fee calculation is preformed correctly when the chain restarts. We got a nice natural test of this in fact when the chain went down.